examples/Gjs/gio-2-cat-ambient/tsconfig.alias.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Please remove the `tsconfig.alias.json` from the repository as it will be generated automatically